src/neo4j_genai/prompts.py

```diff
@@ -0,0 +1,30 @@
+#  Copyright (c) "Neo4j"
+#  Neo4j Sweden AB [https://neo4j.com]
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      https://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+TEXT_TO_CYPHER_PROMPT = """
+Task: Generate a Cypher statement for querying a Neo4j graph database from a user input.
+
+Schema:
+{schema}
+
+Examples (optional):
+{examples}
+
+Input:
+{input}
+
+Do not use any properties or relationships not included in the schema.
+Do not include triple backticks ``` or any additional text except the generated Cypher statement in your response.
```

Comment:
How did you come up with this prompt?

Reply:
I came up with it myself. It was based on the prompts [here](https://github.com/langchain-ai/langchain/blob/09919c2cd5398068c43662ff3acf2f5c21c35747/libs/community/langchain_community/chains/graph_qa/prompts.py#L98). None of those prompts seem quite right however as most request the LLM use the returned records from the Neo4j database to directly respond to the user's input. Given this is a retriever I assume we just want the raw records returned. They also don't allow for the addition of few-shot learning examples, which in LangChain is done through the `PromptTemplate` class. I considered implementing a version of this class here but thought it would be better to keep things simple and just use templated strings for now. I also tried LangChain hub but couldn't find anything useful there.

Reply:
Done!